docs/components/modeler/bpmn/tasks.md

```diff
@@ -4,6 +4,24 @@ title: "Overview"
 description: "This document outlines an overview of supported elements."
 ---
 
+import ReactPlayer from 'react-player'
+
+The basic elements of BPMN processes are tasks; these are atomic units of work composed to create a meaningful result. Whenever a token reaches a task, the token stops and Zeebe creates a job and notifies a registered worker to perform work. When that handler signals completion, the token continues on the outgoing sequence flow.
+
+<center>
+<ReactPlayer
+playing
+loop
+playsinline
+height="300px"
+url={[
+{src: '/videos/tasks.mp4', type: 'video/mp4'}
```

Comment:
Where are these videos from? They look a bit funny. The token simulation visuals are outdated and they're not properly centered.

<img width="312" alt="brave_KgQmYCBKO3" src="https://github.com/camunda/camunda-docs/assets/7633572/5ec5e8ac-30d4-428e-8f24-1b50885e8269">

Compare to the current token simulation visuals:

<img width="418" alt="brave_kdpB9XZ4e5" src="https://github.com/camunda/camunda-docs/assets/7633572/4df68383-3aae-422d-833b-5dfb49622083">

Reply:
Furthermore, the spacing around the videos looks broken.

![image](https://github.com/camunda/camunda-docs/assets/7633572/521f745f-a7fd-4967-8b6d-34a5448ac3d3)

Reply:
This is content from https://docs.camunda.io/docs/components/modeler/bpmn/bpmn-primer/#bpmn-elements (for example) carried over to the individual pages.

I can remove these videos and summarize the content in each element for this issue, but if we need to update all of these, perhaps we should create a subsequent issue to update the videos, or remove them entirely, and substitute screenshots for longevity. Thoughts?

Reply:
Thanks for clarification. Rather than reusing these outdated screen recordings I'd either just remove them or use screenshots which [can easily be automated](https://github.com/camunda/camunda-docs-modeler-screenshots/). We could create an issue for a future enhancement by using screen recordings if we find a way to automate those as well. Otherwise they're impossible to maintain.

Reply:
Removed these in my latest commit and adjusted the wording. Let me know what you think, and I can backport 👍

Reply:
Will have another look today.